tests/test_types.py

```diff
@@ -733,6 +734,34 @@ class Model(BaseModel):
     ]
 
 
+def test_str_limited_good():
+    max_length = 5
+
+    class Model(BaseModel):
+        v: StrLimited[max_length]
+
+    m = Model(v='abcde')
```

Comment:
```suggestion
    m = Model(v='abcde ')
```
show that it also strips white space.